SU-SWS/jumpstart_ui
# Added utility plain list class to remove list markers and spacing

# READY FOR REVIEW

# Summary
- Utility class for lists to remove the markers and spacing.
- Mostly for view lists or field item lists that will be styled in some certain way.

# Need Review By (Date)
- :shrug: 

# Urgency
- low

# Steps to Test
1. checkout this branch
2. create a list of items and add the class `su-plain-list` to the `<ul>` or `<ol>`
3. verify no markers are displayed on the list elements.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
